add walletMonitor env

hiepmai-babylonchain opened the pull request.

hiepmai-babylonchain committed:
add walletMonitor

hiepmai-babylonchain committed:
add circleci

hiepmai-babylonchain committed:
test build

hiepmai-babylonchain committed:
Add function to convert string to bool

filippos47 reviewed (approved):
Very nice!

filippos47 reviewed (commented):
So, wallet monitoring implementation is not compatible with v0.26.0 and we choose to disable it?
Is there any way to make the wallet monitoring compatible with v0.26.0 instead?

hiepmai-babylonchain commented:
The wallet monitor still works in v0.26.0 but only `keyPoolOldestMetric` metric is not available (Legacy wallets only) so I removed them.
https://github.com/babylonchain/bitcoind-exporter/pull/1/files#diff-4a8b61e44513cd3243f5035d2c63edce502b1d13fbaa789a3122cee6d9f3768dL70

I added the `walletMonitor` env to allow us to have the option to run the exporter without the need of wallet creation.

hiepmai-babylonchain committed:
Update branch filter

hiepmai-babylonchain committed:
Update branch filter another place
